### ISSUE TITLE
The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

### ISSUE THREAD

**Issue by goseesomething** [clovaai/donut#255]
Even through I tried to rewrite the def configure_optimizers like following, I still get this problem.

Besides, I am using python 3.8, every other requirements are the same

![image](https://github.com/clovaai/donut/assets/40739576/71e582df-d605-486d-8a0a-3591c672f92d)

![image](https://github.com/clovaai/donut/assets/40739576/57902fa7-41a1-4231-b277-fd1598184268)

Package                 Version
----------------------- ------------
absl-py                 2.0.0
aiohttp                 3.8.5
aiosignal               1.3.1
async-timeout           4.0.3
asynctest               0.13.0
attrs                   23.1.0
cachetools              5.3.1
certifi                 2022.12.7
charset-normalizer      3.2.0
click                   8.1.7
colorama                0.4.6
datasets                2.13.2
dill                    0.3.6
filelock                3.12.2
frozenlist              1.3.3
fsspec                  2023.1.0
google-auth             2.23.0
google-auth-oauthlib    0.4.6
grpcio                  1.58.0
huggingface-hub         0.16.4
idna                    3.4
importlib-metadata      6.7.0
Jinja2                  3.1.2
joblib                  1.3.2
Markdown                3.4.4
MarkupSafe              2.1.3
mpmath                  1.3.0
multidict               6.0.4
multiprocess            0.70.14
munch                   4.0.0
networkx                3.0
nltk                    3.8.1
numpy                   1.21.6
oauthlib                3.2.2
packaging               23.1
pandas                  1.3.5
Pillow                  9.5.0
pip                     23.2.1
protobuf                3.20.1
pyarrow                 12.0.1
pyasn1                  0.5.0
pyasn1-modules          0.3.0
pyDeprecate             0.3.2
python-dateutil         2.8.2
pytorch-lightning       1.6.4
pytz                    2023.3.post1
PyYAML                  6.0.1
regex                   2023.8.8
requests                2.31.0
requests-oauthlib       1.3.1
rsa                     4.9
ruamel.yaml             0.17.32
ruamel.yaml.clib        0.2.7
sacremoses              0.0.53
sconf                   0.2.5
sentencepiece           0.1.99
setuptools              65.6.3
six                     1.16.0
sympy                   1.12
tensorboard             2.11.2
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.1
timm                    0.5.4
tokenizers              0.10.3
torch                   2.0.1+cu117
torchaudio              2.0.2+cu117
torchmetrics            0.11.4
torchvision             0.15.2+cu117
tqdm                    4.66.1
transformers            4.11.3
typing_extensions       4.7.1
urllib3                 1.26.16
Werkzeug                2.2.3
wheel                   0.38.4
wincertstore            0.2
xxhash                  3.3.0
yarl                    1.9.2
zipp                    3.15.0
zss                     1.2.0

**Comment by goseesomething** [comment]
The fix is here.

https://github.com/Lightning-AI/lightning/issues/15912

![image](https://github.com/clovaai/donut/assets/40739576/f57382fc-4158-4ed3-b288-8f958a335214)

**Comment by stanislav215** [comment]
Resolved from https://github.com/Lightning-AI/pytorch-lightning/releases/tag/1.8.4
